config/cmake/Finddftd4-blas.cmake

```diff
@@ -0,0 +1,56 @@
+# This file is part of dftd4.
+# SPDX-Identifier: LGPL-3.0-or-later
+#
+# dftd4 is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# dftd4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with dftd4.  If not, see <https://www.gnu.org/licenses/>.
+
+if ((DFTD4_BLAS MATCHES ^Intel) OR (DEFINED ENV{MKLROOT}))
+  enable_language("C")
+endif()
+
+if(WITH_ILP64)
+  message(STATUS "Using LAPACK/BLAS ILP64 interface")
+  set(BLA_SIZEOF_INTEGER 8)
+  set(_nvpl_int "_ilp64")
+else()
+  set(BLA_SIZEOF_INTEGER 4)
```

Comment:
This was not set in https://github.com/grimme-lab/multicharge/pull/44. Is this required?

Reply:
Good catch! It is usually found by default, but it would indeed be better to be explicit. I'll update `multicharge` to do the same.

Reply:
Fine by me.